app/scripts/controllers/network/provider-api-tests/shared-tests.js

```diff
@@ -115,21 +115,31 @@ export function testsForProviderType(providerType) {
       const notHandledByMiddleware = [
         { name: 'eth_accounts', numberOfParameters: 0 },
         { name: 'eth_coinbase', numberOfParameters: 0 },
+        { name: 'eth_createAccessList', numberOfParameters: 2 },
         { name: 'eth_feeHistory', numberOfParameters: 3 },
         { name: 'eth_getFilterChanges', numberOfParameters: 1 },
         { name: 'eth_getLogs', numberOfParameters: 1 },
+        { name: 'eth_getProof', numberOfParameters: 3 },
         { name: 'eth_getWork', numberOfParameters: 0 },
         { name: 'eth_hashrate', numberOfParameters: 0 },
+        { name: 'eth_maxPriorityFeePerGas', numberOfParameters: 0 },
         { name: 'eth_mining', numberOfParameters: 0 },
         { name: 'eth_newBlockFilter', numberOfParameters: 0 },
         { name: 'eth_newFilter', numberOfParameters: 1 },
         { name: 'eth_newPendingTransactionFilter', numberOfParameters: 0 },
         { name: 'eth_sendRawTransaction', numberOfParameters: 1 },
+        { name: 'eth_signTransaction', numberOfParameters: 1 },
         { name: 'eth_sendTransaction', numberOfParameters: 1 },
         { name: 'eth_sign', numberOfParameters: 2 },
+        { name: 'eth_submitHashRate', numberOfParameters: 2 },
         { name: 'eth_submitWork', numberOfParameters: 3 },
         { name: 'eth_syncing', numberOfParameters: 0 },
         { name: 'eth_uninstallFilter', numberOfParameters: 1 },
+        { name: 'debug_getRawHeader', numberOfParameters: 1 },
```

Comment:
`debug_getRawHeader `, `debug_getRawBlock` and `debug_getRawReceipts` seem to accept a block (block number or tag) as param. Should these ones then fall into the group 'supportingBlockParam'?

Reply:
Good question. So the "supporting block param" group isn't exactly the set of methods that support a block param, it's the set of methods that our middleware recognizes the block parameter for. That's why there are a number of methods in this group and the "assume there is no block param" group that do have block parameters.

As we add support in our middleware for new methods, we'll likely have to move some of these between groups to account for that.